js/scripts.js

```diff
@@ -1 +1,6 @@
- 
\ No newline at end of file
+ var text = "Papugi koszą trawę, unosząc się nad ziemią 15 centymetrów, na czerwonych dywanach.";
+ var animal = "Zielone słonie";
+ var animalUpperCased = animal.toUpperCase();
+ var textAfter = text.replace('Papugi', animalUpperCased);
```

Comment:
Zrób jeszcze tak żeby zamiast Papug, zamieniany był zawsze pierwszy wyraz zmiennej `text` - bez względu na to co tam będzie. Możesz użyć do tego metody `split` (google -> "mdn split")